### ISSUE TITLE
SkipTo removes ignored expression since v3.1.0

### ISSUE THREAD

**Issue by catcombo** [pyparsing/pyparsing#500]
```python
def parse(tokens):
    # pyparsing 3.0.9 -> ['a_*_a']
    # pyparsing 3.1.0 -> ['']
    print(tokens)


wrapper = "a" + SkipTo("a") + "a"
expr = "*" + SkipTo("*", ignore=wrapper).set_parse_action(parse) + "*"

# pyparsing 3.0.9 -> ['*', 'a_*_a', '*']
# pyparsing 3.1.0 -> ['*', '', '*']
print(expr.parse_string("*a_*_a*"))
```
The code example works correctly on `pyparsing` 3.0.9. But since 3.1.0 it's no longer possible to get access to the ignored expression and parse it differently.

**Comment by ptmcg** [comment]
It looks like this broke when fixing #475 - I'll use your test case to work out a fix that addresses both issues.

**Comment by catcombo** [comment]
Thank you!

**Comment by ptmcg** [comment]
Fixed in 3.1.1

**Comment by catcombo** [comment]
Thank you for solving the problem so quickly!
